templates/authors/_entry.twig

```diff
@@ -0,0 +1,45 @@
+{% extends "_layouts/_base" %}
+{% import '_macros/global' as global %}
+
+{% set authorName = entry.firstName ~ ' ' ~ entry.lastName %}
+
+{% set sourceSections = ['articles', 'podcasts', 'events', 'videos', 'courses'] %}
```

Comment:
@johnnymueller FYI: lines 6-9 or a good example of how to mash together the channels and then pull entries from them that are related to something like category or entry.id (could be author, product, etc).